website/docs/getting-started/setup/instance-configuration/authentication/security-assertion-markup-language-saml/auth0.mdx

```diff
@@ -0,0 +1,82 @@
+---
+description: >-
+  Setup SAML using Auth0
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Auth0
+
+To configure Appsmith to use [Auth0](https://auth0.com/) as a SAML provider, follow the steps below:
+
+## Prerequisites
+
+1. In Appsmith, go to **Admin Settings** > **Authentication** and click **Enable** on  **SAML 2.0**.
+
+2. Copy the **Redirect URL** and **Entity URL** from the **SAML 2.0** configuration page to add them later in the Auth0 settings. 
```

Comment:
is the Entity URL getting pasted anywhere?

Reply:
No it's not in Auth0.